app/api/v2/models/user_models.py

```diff
@@ -15,15 +15,13 @@ class UsersModel():
     def __init__(self):
         self.db = init_db()
         self.registered = datetime.now().strftime(("%Y-%m-%d %H:%M:%S"))
-        self.user_id = "user_id"
         self.isAdmin = "isAdmin"
 
     """ save our users and appendthem to the database """
     def save(self, firstname, lastname, othernames, username, email,
              phonenumber, password):
 
         userdata = {
```

Comment:
I think we talked about this last time that you should separate variable names with two one more words with an underscore

Reply:
I will make changes to those that I haven't. My aplogies once again.